### ISSUE TITLE
Release Twisted core 2.3.0

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11239]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#1718|
|Type|task|
|Created|2006-05-14 17:09:34Z|

Try it!

<details><summary>Searchable metadata</summary>

```
trac-id__1718 1718
type__task task
reporter__radix radix
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1147626574000000 1147626574000000
changetime__1147641440000000 1147641440000000
version__ 
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix set status to `closed`|
|-|-|

(In [16815]) Merge the release branch for Twisted core 2.3.0

Author: Release Automation System, radix
Reviewers: radix, exarkun

Fixes [#11239](https://github.com/twisted/twisted/issues/11239)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
